plugins/jobtype/src/azkaban/jobtype/HadoopHiveJob.java

```diff
@@ -267,4 +270,23 @@ private static String getSourcePathFromClass(Class<?> containedClass) {
           .getPath();
     }
   }
+  
+  /**
+   * This cancel method, in addition to the default canceling behavior, also kills the MR jobs launched by Hive
+   * on Hadoop
+   */
+  @Override
+  public void cancel() throws InterruptedException {
+    super.cancel();
+
+    info("Cancel called.  Killing the Hive launched MR jobs on the cluster");
+
+    String azExecId = jobProps.getString("azkaban.flow.execid");
+    final String logFilePath =
+        String.format("%s/_job.%s.%s.log", getWorkingDirectory(), azExecId,
+            getId());
+    info("log file path is: " + logFilePath);
```

Comment:
I thought the log file will be on the Azkaban box until it gets retentioned out? (like in 7 days)
Meanwhile, this is in the cancel method, so should work as long as the file is around during the actual time of cancellation